.coveragerc

```diff
@@ -14,6 +14,9 @@ skip_covered = true
 skip_empty = true
 show_missing = true
 exclude_also =
+  if TYPE_CHECKING    # never executes at runtime
```

Comment:
I'm not sure if coverage.py handles inline comments following regexes that can contain `#` in them, too. Have you checked?